src/exporter.js

```diff
@@ -139,4 +134,21 @@ function exporter (hash, dagService, options, callback) {
       })
```

Comment:
all of these (L123 to L134) can be replaced by just passing the callback as the done function for your async.forEachSeries.